packages/selinux-policy/subject.cil

```diff
@@ -121,7 +127,8 @@
 
 ; Subjects shipped with the OS that should only execute verified code.
 (typeattribute verified_s)
-(typeattributeset verified_s (xor (host_s) (runtime_t mount_t api_t init_t)))
+(typeattributeset verified_s (xor (host_s) (
+  runtime_t mount_t api_t init_t kernel_t)))
```

Comment:
This drops `kernel_t` from the `verified_s` set, so that we can retain the more restrictive rule:
```
(neverallow verified_s mutable_o (files (execute)))
```